### PR TITLE
Reuse existing manifest option

### PR DESCRIPTION
**Pull request by monsonjeremy**
Piggy-backing off the work done in this [PR](https://github.com/phamann/rollup-plugin-hash/pull/11) to reuse the existing manifest. I also added this as an optional flag since it's not desired for all users of this plugin.

### CODE REVIEW

**Comment by monsonjeremy** [comment]
@phamann I hope you don't mind but i'm going to publish my own branched version under a new namespace until this is merged because it's a little time sensitive for me. If there is an issue i'm willing to unpublish.

**Comment by GerkinDev** [comment]
It would be good to see it merged, along with other working PRs. I've reviewed this one and I'm OK for it, if you care about my opinion. The PR is very targeted & simply working well.

**Comment by monsonjeremy** [comment]
@phamann could we get a quick approval or a review? I'd like to get this PR off my PR list.

**Comment by monsonjeremy** [comment]
@GerkinDev By the way, if you want to use a forked version with this code I published a fork here: https://www.npmjs.com/package/rollup-plugin-hash-manifest

**Comment by halfzebra** [comment]
@monsonjeremy Hi Jeremy, thanks for implementing this! 👍

File: test/index.js
Comment (GerkinDev) [inline test/index.js]: Useless chore.

File: test/index.js
Comment (GerkinDev) [inline test/index.js]: Useless chore.

**Review by GerkinDev — APPROVED** [review]
Tests case are OK for main cases.

:+1: 

> PS: Sorry for the delay, I still hope this will get merged.

File: test/index.js
Comment (monsonjeremy) [reply test/index.js]: Not sure what you mean here.

Comment (GerkinDev) [reply test/index.js]: I don't see what improvements or changes this diff does. `require` already parses the JSON.

File: test/index.js
Comment (GerkinDev) [reply test/index.js]: Same here

File: test/index.js
Comment (monsonjeremy) [reply test/index.js]: @GerkinDev I've addressed the comments on this PR.

Comment (monsonjeremy) [reply test/index.js]: @GerkinDev, looks like this wasn't a useless chore. Without these, the tests fail on Node 4 since require likely does not parse it in that old node version.

Comment (GerkinDev) [reply test/index.js]: Okay my bad. But since you changed tested node env (see [this diff](https://github.com/phamann/rollup-plugin-hash/pull/23/commits/e29f6ea2fb360387cf3c323eef175f26ab262500)), is it still relevant ?

Comment (monsonjeremy) [reply test/index.js]: Good call, removed those changes. Should be good to merge now!

**Comment by monsonjeremy** [comment]
@GerkinDev I've made all the requested changes, do you mind merging this?

**Comment by GerkinDev** [comment]
@monsonjeremy I'm not a maintainer of this package ^^ I just reviewed to help, and validated your changes. I really can't do anything more. @phamann ping !

**Comment by monsonjeremy** [comment]
@GerkinDev Sorry, thought you were a maintainer! Anyways, i'll give it another week and then i'm going to delete the PR.
@phamann tick tock 😄

**Comment by GerkinDev** [comment]
Dammit.

**Pull request closed without merging**
